x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+API_KEY_ENCODED=$1
+API_ENDPOINT=$2
+
+updateStepProgress() {
+  echo "  GET $API_ENDPOINT/step/$1?status=$2"
+  curl --request GET \
+    --url "$API_ENDPOINT/step/$1?status=$2" \
+    --header "Authorization: ApiKey $API_KEY_ENCODED" \
+    --header "Content-Type: application/json" \
+    --header "kbn-xsrf: true"
+  echo ""
+}
+
+echo "Downloading Elastic Agent"
+# https://www.elastic.co/guide/en/fleet/8.7/install-standalone-elastic-agent.html
+curl -L -O https://artifacts.elastic.co/downloads/beats/elastic-agent/elastic-agent-8.7.1-linux-x86_64.tar.gz
```

Comment:
Ideally the agent version becomes an input param so users always get the version compatible with their Stack version.